packages/common/core-interfaces/api-report/core-interfaces.api.md

```diff
@@ -296,6 +296,12 @@ export interface ILoggingError extends Error {
     getTelemetryProperties(): ITelemetryBaseProperties;
 }
 
+// @public (undocumented)
+export interface Internal_InterfaceOfJsonableTypesWith<T> {
```

Comment:
Adding a public interface named starting with `Internal_` seems questionable. If our public interface extends this, then I think we should either collapse the two, or name this with better semantics. It should also be documented.

Reply:
There isn't a great reason to even have this type. It is there because the type test code has trouble. We should fix the type test code.